lib/i18n/hygiene/variable_checker.rb

```diff
@@ -11,48 +11,24 @@ def initialize(key, i18n_wrapper, primary_locale, locales = [])
         @locales = locales
       end
 
-      def mismatched_variables_found?
+      def mismatched_variables
         @locales.each do |locale|
```

Comment:
Now that this has been reduced to a single statement within the block, would it be even simpler to change this to use the single line form?

```ruby
@locales.each { |locale|  yield(locale, @key, missing_variables(locale)) }
```